clustered/client/src/main/java/org/ehcache/clustered/client/config/ClusteringServiceConfiguration.java

```diff
@@ -60,6 +60,14 @@ public ClusteringServiceConfiguration(URI clusterUri) {
     this(clusterUri, Timeouts.DEFAULT);
   }
 
+  /**
+   * Creates a {@code ClusteringServiceConfiguration} from the properties provided.
+   *
+   * @param servers the non-{@code null} Iterable<InetSocketAddress> identifying the servers in the cluster
```

Comment:
I think stating the type like that is a little verbose maybe make it a little more prose like: `an iterable of the servers in the cluster`?

Reply:
Done